src/nora/components/Example/Example.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
```

Comment:
These are just intended to be placeholder components to be replaced by "real ones" once we start to actually use these